snare/tanner_handler.py

```diff
@@ -0,0 +1,139 @@
+import asyncio
```

Comment:
unused import

Reply:
I think we have used asyncio in all `async def` syntax. What do you think ?